backend/update_requirements.sh

```diff
@@ -3,7 +3,7 @@
 # Usage: update_requirements.sh <requirements.in >requirements.txt
 
 set -euo pipefail
-IMAGE=${1:-"python:3.5"}
+IMAGE=${1:-"python:3.6"}
```

Comment:
Did you run this script to the `requirements.in` files? I believe many of them will update what's in `requirements.txt`, because a lot of library versions depend on python version.